codegen.yml

```diff
@@ -17,4 +17,7 @@ config:
   namingConvention:
     enumValues: keep
   scalars:
-    BigInt: BigNumber
\ No newline at end of file
+    BigInt: BigNumber
+  avoidOptionals:
+    field: true
+  skipTypename: true
```

Comment:
- `avoidOptionals` prevents typing some things as `undefined` that in practice will only ever be defined or `null`
- `skipTypename` avoids expecting a `typename: string` property in Apollo responses, which creates some type compatibility issues

Reply:
ay